### ISSUE TITLE
None Followers

### ISSUE THREAD

**Issue by FireMasterK** [pluja/Parasitter#120]
Example URL: https://yotter.xyz/c/UCYEK6xds6eo-3tr4xRdflmQ
![image](https://user-images.githubusercontent.com/20838718/96730828-39e16780-13d4-11eb-8bb0-a8224ee2c9d1.png)

**Comment by pluja** [comment]
This is an issue with the data extraction and I'm not being able to extract the Followers. Maybe we'll need to remove the followers.

**Comment by FireMasterK** [comment]
Looks like the subscriber count is no longer visible in the channel page, was this intentional?

**Comment by pluja** [comment]
Yes, as I can't retrieve the followers info (for now) I thought it was better not to show the info rather than showing 'NONE' always.
